### PR TITLE
Allow selling CELO for cEUR even if user doesnt have a balance

### LINKED ISSUE #865: Sell CELO screen has the wrong title.
When inputting the amount during the Sell CELO flow, the screen is titled "Buy CELO".
It should be titled "Sell CELO"

### LINKED ISSUE #872: Always show balance dropdown in the buy/sell Celo flow
Originally we have discussed that exchanging Celo to cEUR will be the only way to get cEUR (assuming that there is now way to cash in in cEUR directly). But now it is not possible. Should 

To do: the dropdown to switch between cUSD/cEUR balances should always be on the screen in Celo exchange flow. Even if the user doesn't own any cEUR.

### PR DESCRIPTION
**Pull request by gnardini**
### Description

When a user doesn't have a cEUR balance, if they try to sell CELO they can now sell the CELO for cEUR as well.

### Other changes

The issue with the title not saying 'Sell CELO' is also fixed with this change.

### Tested

Manually

### How others should test

With an account that has no cEUR balance, go to the CELO tab and press `Sell`. See that you can tap on the header so that you sell CELO for cEUR.

### Related issues

- Fixes #872
- Fixes #865

### Backwards compatibility

N/A

### CODE REVIEW

File: packages/mobile/src/exchange/ExchangeScreenHeader.tsx [hunk 1]
```diff
@@ -43,9 +42,9 @@ function ExchangeTradeScreenHeader({ currency, makerToken, onChangeCurrency }: P
 
     let titleText
     let title
-    const singleTokenAvailable = currenciesWithBalance < 2
-    if (singleTokenAvailable) {
-      title = isCeloPurchase ? i18n.t('exchangeFlow9:buyGold') : i18n.t('exchangeFlow9:sellGold')
+    const tokenPickerEnabled = currenciesWithBalance >= 2 || !isCeloPurchase
+    if (!tokenPickerEnabled) {
+      title = i18n.t('exchangeFlow9:buyGold')
```
Comment (tarikbellamine) [inline packages/mobile/src/exchange/ExchangeScreenHeader.tsx:47]: I'm a little confused by this...is there never a time where `title` should equal `exchangeFlow9:buyGold`?

**Review by tarikbellamine — APPROVED** [review]
Left a minor question but looks good otherwise!

File: packages/mobile/src/exchange/ExchangeScreenHeader.tsx (hunk 1 above)
Comment (gnardini) [reply packages/mobile/src/exchange/ExchangeScreenHeader.tsx:47]: For selling CELO we always show both currencies no matter what, and when we show both currencies we don't show the buy/sell CELO text, the header shows up like this:

Screen Shot 2021-07-12 at 15 07 10

so in no cases does the title say `exchangeFlow9:sellGold` and it only says `exchangeFlow9:buyGold` when the user is buying CELO and they have a balance in only one currency.

Comment (tarikbellamine) [reply packages/mobile/src/exchange/ExchangeScreenHeader.tsx:47]: Got it. Can we delete `exchangeFlow9:buyGold`?

Comment (gnardini) [reply packages/mobile/src/exchange/ExchangeScreenHeader.tsx:47]: It's still used in the header of the `ExchangeReviewScreen`

**Comment by Celoqa** [comment]
Hi @MuckT We have verified this issue on Latest Android internal build V1.17.0 & Test flight build V1.17.0 and updated the status in test quality tool.

**Test quality tool link:**
 
- **Android:**
       
  -  https://valora.testquality.com/project/12128/plan/21240/run/149557/result/2450226

- **iOS:**
     

    - https://valora.testquality.com/project/12128/plan/21240/run/149557/result/2450217

**Verified On Devices:** iPhone SE (14.2), Samsung Galaxy S4 (4.4.2), 

CC: @gnardini

**Comment by ValoraQA** [comment]
Hi @MuckT We have verified this issue on Latest Android internal build V1.18.0  and updated the status in test quality tool.

Test quality tool link:

Android (11):
https://valora.testquality.com/project/12128/plan/21657/run/149950/result/2469325

**Verified on Devices:** Google Pixel 4a (11.0)

**Comment by ValoraQA** [comment]
Hi @MuckT We have verified this issue on Latest Android internal build V1.18.0 and updated the status in test quality tool.

Test quality tool link:

Android (10):
https://valora.testquality.com/project/12128/plan/21657/run/149950/result/2469314

**Verified on Devices:** Redmi Note 8 (10.0)

CC: @gnardini
Thanks..!!

**Comment by ValoraQA** [comment]
Hey @MuckT  We have verified this issue on Latest  Test Flight build V1.18.0 and updated the status in test quality tool.

**Test quality tool link:**
- https://valora.testquality.com/project/12128/plan/21657/run/149950/result/2469270

**Verified on Devices:** iPhone SE (13.6.1)
CC: @gnardini 
Thanks..!!!

**Pull request merged**